src/lib/components/Circos.react.js

```diff
@@ -173,32 +173,34 @@ export default class Circos extends Component {
 
     configCircos(layout, config, tracks, setProps) {
         this.circos.layout(layout, config);
-        tracks.forEach((track, index) => {
-            const {id, data, config, type} = track;
+        if (tracks) {
+            tracks.forEach((track, index) => {
+                const {id, data, config, type} = track;
 
-            // Since config is const, can't manipulate and throws error
-            let configApply;
+                // Since config is const, can't manipulate and throws error
+                let configApply;
 
-            if (typeof config !== 'undefined') {
-                configApply = config;
+                if (typeof config !== 'undefined') {
```

Comment:
is es-lint preventing you from using ```undefined```?

Reply:
Yep: 

```
  183:32  error  Unexpected use of undefined  no-undefined
```

Is it better to disable that error on this line?

Reply:
Yep: 

```
  183:32  error  Unexpected use of undefined  no-undefined
```

Is it better to disable that error on this line?

Reply:
This eslint rule exists to prevent doing checks against `undefined`. This is only relevant if targeting browsers supporting ES3 (which we are not, and never will do) -- all moderately modern browsers fully support the ES5 standard which guarantees that `undefined` is read-only.

In ES3 you could do something like `undefined = 'potato'` and all subsequent checks against it would have been, well, potato..

You can remove this rule as it is useless. We've done the same, for the same reasons not long ago in the renderer.

Reply:
Fixed in e17d9b0.